components/Common/Select/index.module.css

```
@@ -59,6 +59,8 @@
 }
 
 .dropdown {
+  @apply max-h-48
+    overflow-y-auto;
   @apply max-w-xs
```

```suggestion
  @apply max-h-48
    overflow-y-auto
    max-w-xs
```

You can simplify doing that. Also don't miss to `npx turbo format`